app/assets/javascripts/discourse/app/widgets/post-stream.js

```diff
@@ -276,6 +278,18 @@ export default createWidget("post-stream", {
         );
       }
 
+      if (
+        i !== postArrayLength - 1 &&
+        maxPostNumber <= attrs.highestPostNumber &&
```

Comment:
This is to prevent the last visit line from shifting as new posts come in while the user is on the page.